### DISCUSSION TITLE
SendGrid action doesn't trigger as desired.

### DISCUSSION THREAD

**Discussion by BeeBombshell** [AntlerVC/firetable#917] [Support / Q&A] [answered]
In reference to a discussion on GitHub:

I'm trying to get sendGrid running on updating a row. I followed the instructions, but when updating a row, nothing happens - also not in the activity-list of sendGrid. 
The SendGrid-test works; the sendgrid-key is in the Security-Manager; the templateID is in the code.
Where do I continue debugging this?

**Comment by webwurm** [comment, marked as answer]
Okay, I didn't solve it as Extension now (like [here](https://docs.rowy.io/extensions/sendgrid-email)), but instead as action like [here](https://docs.rowy.io/field-types/action#sending-an-email-via-sengrid). That of course doesn't solve the root of the problem.

**Reply by harinij** [reply]
Hi @webwurm for the extension, a few things to check are make sure the extension is enabled and deployed.

<img width="294" alt="Screen Shot 2022-11-08 at 5 17 28 pm" src="https://user-images.githubusercontent.com/307298/200712515-36026b2a-3fdc-4968-aacc-1d5b49422c55.png">

You can also look at the cloud logs here 
<img width="302" alt="Screen Shot 2022-11-08 at 5 17 48 pm" src="https://user-images.githubusercontent.com/307298/200713237-05135f3b-3946-46a4-a5a2-4081948110d7.png">

If the build and deploy was successful, then the function section of cloud logs should show you relevant info on why it didn't work. Let us know if you are still not able to see why, and we can hop on a call to go over it together.

**Reply by webwurm** [reply]
Well, it was enabled and deployed. 
But thanks a lot, the info with the cloud logs is very important, I didn't know that.
Otherwise I implemented it as action now, because that actually fits better to my case - and it works flawlessly.
Thanks again!
